__tests__/__snapshots__/authutil.test.ts.snap

```diff
@@ -1,31 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
i never understood the snapshot thing. Nice to see it deleted :) one less thing to learn